untar/untar.go

```diff
@@ -17,9 +17,35 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"sync"
 	"time"
 )
 
+var (
+	maximumArchiveSize int = 1 << (10 * 3) // 1Gb
```

Comment:
Added 1Gb to decrease the likelihood of breaking existing usage, but it feels too large for a default value.
Keen to here opinions on better values here.